manager/allocator/networkallocator/drivers_network_linux.go

```diff
@@ -0,0 +1,29 @@
+package networkallocator
+
+import (
+	"github.com/docker/libnetwork/drivers/bridge/brmanager"
+	"github.com/docker/libnetwork/drivers/host"
+	"github.com/docker/libnetwork/drivers/ipvlan/ivmanager"
+	"github.com/docker/libnetwork/drivers/macvlan/mvmanager"
+	"github.com/docker/libnetwork/drivers/overlay/ovmanager"
+	"github.com/docker/libnetwork/drivers/remote"
+)
+
+func getInitializers() []initializer {
```

Comment:
same comment: no `get` here.

Also, this could probably just be a `const` in the file